scripts/exregional_run_fcst.sh

```diff
@@ -896,11 +877,13 @@ if [ $(boolify ${WRITE_DOPOST}) = "TRUE" ]; then
       fhr_d=${fhr}
     fi
 
+    post_output_domain=$(uw config realize -i $GLOBAL_VAR_DEFNS_FP --output-format yaml \
+    --key-path=task_run_post.post_output_domain_name)
```

Comment:
```suggestion
    --key-path task_run_post.post_output_domain_name)
```

Based on docs and other code I don't think the `=` is needed